boruta/1-click-boruta/script.whizzml

```diff
@@ -0,0 +1,386 @@
+;; This will eventually be packed into a library
+
+(define IMPORTANCE_INC 1)
+(define IMPORTANCE_DEC -1)
+
+;; choosable-objective-ids
+;;
+;; List of IDs of the fields in the dataset that can be chosen as objective
+;; field.
+;;
+;; Inputs:
+;;  fields: (map) Fields structure
+;; Output: (list) list of field IDs
+(define (choosable-objective-ids fields)
+  (let (field-val (lambda (fid k) (get-in fields [fid k]))
```

Comment:
`(fields [fid k])` ... and all `get-in` and `get` elsewhere can be changed too...